mcp-server-elevenlabs/src/mcp_server_elevenlabs/schemas.py

```
@@ -0,0 +1,53 @@
+        ),
+    )
+
+
+class GenerateVoiceResponse(BaseModel):
+    success: bool
+    filename: str
+    file_path: str
+    media_type: str
+    download_url: str
+    audio_bytes: int
+    audio_base64: Optional[str] = None
```

Ruff (0.14.14)

[warning] No newline at end of file

Add trailing newline

(W292)